infrastructure/modules/aks-resources/linkerd.tf

```diff
@@ -0,0 +1,52 @@
+resource "azapi_resource" "linkerd" {
+  depends_on = [azapi_resource.cert_manager]
+  type       = "Microsoft.KubernetesConfiguration/fluxConfigurations@2024-11-01"
+  name       = "linkerd"
+  parent_id  = var.azurerm_kubernetes_cluster_id
+  body = {
+    properties = {
+      kustomizations = {
+        linkerd = {
+          force = false
+          path  = "./"
+          postBuild = {
+            substitute = {
+              DISABLE_IPV6 = "false"
+            }
+          }
+          prune                  = false
+          retryIntervalInSeconds = 300
+          syncIntervalInSeconds  = 300
+          timeoutInSeconds       = 600
+          wait                   = true
+        },
+        linkerd-post-deploy = {
+          dependsOn = [
+            "linkerd"
+          ]
+          force                  = true
+          path                   = "./post-deploy/"
+          prune                  = true
+          retryIntervalInSeconds = 300
+          syncIntervalInSeconds  = 31557600
```

Comment:
I know this is to prevent it from running on other times than deploy, but should we add a comment on the end of the line so that the tribal knowledge is documented